Dockerfile_wpa.template

```diff
@@ -0,0 +1,43 @@
+# fog-sw BUILDER
+FROM ubuntu:20.04 as fog-sw-builder
+
+ARG BUILD_NUMBER
+ARG DISTRIBUTION
+ARG ARCHITECTURE
+ARG COMMIT_ID
+ARG GIT_VER
+ARG PACKAGE_SUBDIR
+ARG ROS
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install build dependencies
+RUN apt-get update -y && apt-get install -y --no-install-recommends \
+    curl \
+    build-essential \
+    dh-make debhelper \
+    cmake \
+    git-core \
+    fakeroot \
+    python3-bloom \
+    dpkg-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+### INCLUDE_DEPENDENCIES
+
+WORKDIR /build
+
+COPY . .
+
+RUN params="-m $(realpath .) " \
+    && [ ! "${BUILD_NUMBER}" = "" ] && params="$params -b ${BUILD_NUMBER}" || : \
+    && [ ! "${DISTRIBUTION}" = "" ] && params="$params -d ${DISTRIBUTION}" || : \
+    && [ ! "${ARCHITECTURE}" = "" ] && params="$params -a ${ARCHITECTURE}" || : \
+    && [ ! "${COMMIT_ID}" = "" ] && params="$params -c ${COMMIT_ID}" || : \
+    && [ ! "${GIT_VER}" = "" ] && params="$params -g ${GIT_VER}" || : \
+    && [ ! "${PACKAGE_SUBDIR}" = "" ] && params="$params -s ${PACKAGE_SUBDIR}" || : \
+    && [ ! "${ROS}" = "" ] && params="$params -r" || : \
+    && ./package_wpa.sh $params
```

Comment:
package_wpa.sh missing

Reply:
it is in wpa repo: https://github.com/tiiuae/wpa/tree/DP-796

Reply:
or branch in repo